platform/ios/src/MGLPinAnnotationView.h

```diff
@@ -0,0 +1,12 @@
+#import <Mapbox/Mapbox.h>
+
+MGL_EXPORT
+
+@interface MGLPinAnnotationView : MGLAnnotationView
+
+@property(nonatomic) UIColor *shadowColor;
+@property(nonatomic) UIColor *pinColor;
+@property(nonatomic) UIColor *strokeColor;
+@property(nonatomic) UIColor *innerColor;
```

Comment:
Would it be possible to reduce the number of color properties, perhaps by automatically choosing stroke and inner colors that contrast well with the pin color? That way, the class will be more future-proof, in case we change the design of the default annotation image and therefore need to adjust this view’s appearance.

Reply:
I would like to keep the number of configureable colors the same in order to keep the class more customizable. Allowing the developer to select their own stroke and inner pin colors will make this class more appealing to use.